contracts/adapters/Distribute.sol

```diff
@@ -193,15 +193,20 @@ contract DistributeContract is IDistribute, DaoConstants, MemberGuard {
 
         // Checks if the proposal has passed.
         IVoting votingContract = IVoting(dao.getAdapterAddress(VOTING));
-        require(
-            votingContract.voteResult(dao, proposalId) ==
-                IVoting.VotingState.PASS,
-            "proposal did not pass"
-        );
-
-        distribution.status = DistributionStatus.IN_PROGRESS;
-        distribution.blockNumber = block.number;
-        ongoingDistributions[address(dao)] = proposalId;
+        IVoting.VotingState voteResult =
+            votingContract.voteResult(dao, proposalId);
+        if (voteResult == IVoting.VotingState.PASS) {
+            distribution.status = DistributionStatus.IN_PROGRESS;
+            distribution.blockNumber = block.number;
+            ongoingDistributions[address(dao)] = proposalId;
+        } else if (
+            voteResult == IVoting.VotingState.NOT_PASS ||
+            voteResult == IVoting.VotingState.TIE
+        ) {
+            distribution.status = DistributionStatus.FAILED;
```

Comment:
I would also call dao.processProposal(proposalId) here to make it as processed.

Reply:
it was called on line 175 :)